allennlp/models/bidirectional_lm.py

```diff
@@ -94,211 +58,12 @@ def __init__(self,
                  num_samples: int = None,
                  sparse_embeddings: bool = False,
                  initializer: InitializerApplicator = None) -> None:
```

Comment:
It is possible to not duplicate the input arguments from `LanguageModel` and just use `*args, **kwargs` so changes in `LanguageModel` are automatically reflected here?  Or does that break `from_params`?

Reply:
that would be great if it worked, i'll try it out. thanks for the suggestion